frontend/src/global_styles/content/work_packages/tabs/_tab_content.sass

```diff
@@ -28,15 +28,19 @@
 
 .op-tab-content
   &--container
-    .spot-list:not(:last-child)
-      margin-bottom: $spot-spacing-0-75
-
     .op-files-tab--storage-info-box:not(:last-child)
       margin-bottom: $spot-spacing-1
 
   &--tab-section
     margin-bottom: $spot-spacing-2
 
+    &_no-header
```

Comment:
I'd move this whole block to the `_files.sass` as it those are file-specific rules and `tab_content` specific

Reply:
Alright, I was thinking about putting it into the `tab_content`, as the `_no-header` modifier belongs to a section, which is not files tab specific.

But there is no issue with moving it, will do.

Reply:
Thanks 👍 It is fine to merge from my side afterwards